app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -78,6 +79,17 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+    - detect = "detectClicked({target: '.detect_button'})"
+    %span
+      %miq-button.detect_button{"ng-if" => detection ? 'true' : 'false',
+                  :name            => _("Detect"),
+                  "on-click"       => detect,
+                  :enabled         => "isDetectionEnabled()",
+                  "disabled-title" => _("Required information missing"),
+                  "enabled-title"  => _("Detect Endpoint"),
+                  :xs              => 'true',
+                  :primary         => 'true'}
+
       %span.help-block{"ng-show" => "angularForm.#{prefix}_hostname.$error.required"}
```

Comment:
(these error `%span`s need to be siblings of the `%input`, not cousins) (https://github.com/ManageIQ/manageiq-ui-classic/pull/1304#issuecomment-306461279)